assets/js/base/context/hooks/shipping/use-shipping-data.ts

```diff
@@ -99,10 +99,8 @@ export const useShippingData = (): ShippingData => {
 			 *
 			 * Forces pickup location to be selected for all packages since we don't allow a mix of shipping and pickup.
 			 */
-			if (
-				hasCollectableRate( newShippingRateId.split( ':' )[ 0 ] ) ||
-				hasSelectedLocalPickup
-			) {
+			if ( hasCollectableRate( newShippingRateId.split( ':' )[ 0 ] ) ) {
+				// Do not pass a packageId because we want this pickup rate to apply to all packages.
```

Comment:
When `hasSelectedLocalPickup` was in the condition, if one package had local pickup, then changing the rate of one package to a regular shipping rate would change the rate for all packages. Is this desired? If so we can add it back